src/ChlorisEasyAdminBundle/Controller/EasyAdminUserController.php

```diff
@@ -1,14 +1,14 @@
 <?php
 
-namespace AppBundle\Controller\Admin;
+namespace ChlorisEasyAdminBundle\Controller;
 
 use Doctrine\DBAL\Exception\ForeignKeyConstraintViolationException;
 use JavierEguiluz\Bundle\EasyAdminBundle\Controller\AdminController as BaseAdminController;
 use JavierEguiluz\Bundle\EasyAdminBundle\Event\EasyAdminEvents;
 use JavierEguiluz\Bundle\EasyAdminBundle\Exception\EntityRemoveException;
 use Symfony\Component\HttpFoundation\RedirectResponse;
 
-class UserController extends BaseAdminController
+class EasyAdminUserController extends BaseAdminController
```

Comment:
This name tells nothing. Better: `UserControllerPreventingFromSelfDelete` or something like this